tests/phpunit/tests/blocks/getBlockAssetUrl.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Tests for block asset urls.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ *
+ * @since 6.4.0
+ *
+ * @group blocks
+ * @covers ::get_block_asset_url
+ */
+class Tests_Get_Block_Asset_Url extends WP_UnitTestCase {
+	/**
+	 * Original theme directory.
+	 *
+	 * @var string[]
+	 */
+	private $orig_theme_dir;
+
+	public function set_up() {
```

Comment:
Do these need to run before/after each method, or just once for all methods in this class? If the latter, consider using `set_up_before_class()`/`tear_down_after_class()` instead.

Reply:
@joemcgill This is copied from 

https://github.com/WordPress/wordpress-develop/blob/4ae2d1a533c6f6dc3b7d82505763dc732924e0f0/tests/phpunit/tests/theme.php#L27-L58

Reply:
That doesn't answer the question. What purpose are they serving here?